e2e/docs/.vuepress/theme/node/e2eTheme.ts

```diff
@@ -1,26 +1,24 @@
-import type { Page, Theme } from 'vuepress/core'
+import type { Theme } from 'vuepress/core'
 import { getDirname, path } from 'vuepress/utils'
 
-const __dirname = getDirname(import.meta.url)
+const DIRNAME = getDirname(import.meta.url)
```

Comment:
Just curious, I think the original one is fine, and I have seen other repos doing this.

Is there some rules that disallow this?

Reply:
It's all fine. Just don't want to add complicated `@typescript-eslint/naming-convention` override 😮‍💨

Reply:
@Mister-Hope Reverted. It should have been supported in eslint-config-vuepress@5.1.4